notebooks/official/pipelines/custom_tabular_train_batch_pred_bq_pipeline.ipynb

```diff
@@ -687,9 +574,9 @@
       },
       "source": [
         "### Create the trainer task\n",
-        "Within `trainer/`, create a module named `task.py` that serves as the entrypoint for your training code.\n",
+        "Within trainer/, create a module named task.py that serves as the entrypoint for your training code.\n",
         "\n",
-        "The trainer code below preprocesses the train set and store the preprocessing transforms in a scikit-learn pipeline. Further, a [Random Forest model is trained](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html) on the preprocessed train data and added as an estimator to the pipeline. After saving the model, it is uploaded to the Cloud Storage bucket for deployment.\n",
+        "The trainer code below preprocesses the train set and store the preprocessing transforms in a scikit-learn pipeline. Further, a [Random Forest model is trained](https://scikitlearn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html) on the preprocessed train data and added as an estimator to the pipeline. After saving the model, it's uploaded to the Cloud Storage bucket for deployment.\n",
```

Comment:
"store" -> "stores":

- The trainer code below preprocesses the train set and stores the preprocessing transforms in a scikit-learn pipeline.